IBF-Typhoon-model/scr/typhoon_model/pipeline.py

```diff
@@ -71,12 +71,15 @@ def main(path,remote_directory,typhoonname):
     print(str(datetime.now()))
     Activetyphoon=Check_for_active_typhoon.check_active_typhoon()
     if Activetyphoon==[]:
-      remote_dir=remote_directory#'20210421120000' #for downloading test data otherwise set it to None
+      remote_dir='20210421120000' #for downloading test data otherwise set it to None
       Activetyphoon=[typhoonname]  #name of typhoon for test
       logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
+    elif remote_directory!=None:
+      remote_dir='20210421120000' #for downloading test data otherwise set it to None
```

Comment:
I don't quite understand what behaviour you want from the command line arguments. With this line, if someone passes a remote directory to the command line, it would be overwritten

Reply:
then we also have to get the name of the typhoon as an argument for now my thinking is if the user enter any directory name we then force the directory name to that of SURIGAE

Reply:
But with this current logic it makes it impossible to set the remote_dir as an argument, because it will always be overridden

Reply:
Here is some logic that will work:
```python
    TEST_REMOTE_DIR = '20210421120000'
    remote_dir = remote_directory
    if Activetyphoon==[]:
        logging.info(f"No active typhoon in PAR runing for typhoon{typhoonname}")
        # If there is no ative Typhoon and the user has not provided a dir,
        # then set it to the test case
        if remote_dir is None:
            remote_dir = TEST_REMOTE_DIR
        Activetyphoon = [typhoonname]
    elif remote_dir is not None:
        # If there is an active typhoon but the user specified a different one
        logging.info(f"There is an active typhoon, but the user has requested to run for typhoon{typhoonname}")
        Activetyphoon = [typhoonname]
    else:
        logging.info(f"Running on active Typhoon(s) {Activetyphoon}")
```